src/status_im/utils/money.cljs

```diff
@@ -90,6 +90,32 @@
   (when-let [bn (bignumber n)]
     (.dividedBy bn (bignumber (from-decimal decimals)))))
 
+(defn unit->token [n decimals]
+  (when-let [bn (bignumber n)]
+    (.times bn (bignumber (from-decimal decimals)))))
+
+;;NOTE(goranjovic) - We have two basic representations of values that refer to cryptocurrency amounts: formatted and
+;; internal. Formatted representation is the one we show on screens and include in reports, whereas internal
+;; representation is the one that we pass on to ethereum network for execution, transfer, etc.
+;; The difference between the two depends on the number of decimals, i.e. internal representation is expressed in terms
+;; of a whole number of smallest divisible parts of the formatted value.
+;;
+;; E.g. for Ether, it's smallest part is wei or 10^(-18) of 1 ether
```

Comment:
Got it. So it's wei except wei don't exist for tokens :)

Reply:
It's wei if token has 18 decimals. It's cents if it has two decimals, etc...

Reply:
❤️ how this note explains the code below!